profile/README.md

```diff
@@ -1,18 +1,7 @@
-## Hi there 👋
+<img src="https://raw.githubusercontent.com/cdfoundation/artwork/main/cdevents/horizontal/color/cdevents_horizontal-color.svg" alt="CDEvents logo" width="300"/>
 
-This is the home of the events protocol CDEvents and its related tools
+A common specification for Continuous Delivery events. 
 
-CDEvents is currently governed by the [Special Interest Group Events](https://github.com/cdfoundation/sig-events) within the [CD Foundation](https://cd.foundation/)
+CDEvents is currently governed by the [Special Interest Group Events](https://github.com/cdfoundation/sig-events) within the [CD Foundation](https://cd.foundation/).
 
-If you have a general issue for the cdevents organization it could be placed in the [spec repo](https://github.com/cdevents/spec)
-
-<!--
-
-**Here are some ideas to get you started:**
-
-🙋‍♀️ A short introduction - what is your organization all about?
-🌈 Contribution guidelines - how can the community get involved?
-👩‍💻 Useful resources - where can the community find your docs? Is there anything else the community should know?
-🍿 Fun facts - what does your team eat for breakfast?
-🧙 Remember, you can do mighty things with the power of [Markdown](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
--->
+[Contributing](https://github.com/cdevents/spec/issues) | [Get in touch](https://github.com/cdfoundation/sig-events#communication)
```

Comment:
Don't know if this is good but I would like to keep the information in the old document about where to place general issues like creation of a new repo.
```suggestion
[Contributing and all types of general issues](https://github.com/cdevents/spec/issues) | [Get in touch](https://github.com/cdfoundation/sig-events#communication)
```

Reply:
The reason I proposed to remove that from the org profile is that github.com/cdevents is where people new to cdevents are most likely to land for now and org level issues didn't seem relevant for them. 
If someone is interested in joining the org or creating a new issue I think it would be good for them to reach out anyways, since we don't have processes in place yet :P 

What about the following format?
[Contributing](https://github.com/cdevents/spec/issues) | [Requests & Issues]((https://github.com/cdevents/spec/issues)) | [Get in touch](https://github.com/cdfoundation/sig-events#communication)

Reply:
I get your point, but where should we document how to create new repos and such?

Reply:
I don't expect creating new repos is something that will happen very often, and it's not something that a new comer would need to do? If someone is interested in creating a repo in the cdevents organization, I think the "Report and issue" or "Get in touch" links should provide enough guidance. 

I'm not convinced that "Reporting a general issue" will provide better guidance than "Reporting an issue" :)

Reply:
@m-linner-ericsson Updated now with 
```
[Contributing](https://github.com/cdevents/spec/issues) | [Get in touch](https://github.com/cdfoundation/sig-events#communication) | [General Issues](https://github.com/cdevents/spec/issues)
```

Does this look ok?